app/models/motorcycle.rb

```diff
@@ -0,0 +1,8 @@
+class Motorcycle < ApplicationRecord
+  belongs_to :user, class_name: 'User'
```

Comment:
I think motorcycles can have many user to reserve them. You can change this line to:
`has_many :users, through: :reservations`

Reply:
@karayamanemre  Are we going to refer how many users a motorcyle have?

Reply:
I don't think we need to refer. @NatanimA